Sources/Sentry/SentryPerformanceTracker.m

```diff
@@ -213,7 +215,12 @@ - (void)finishSpan:(SentrySpanId *)spanId withStatus:(SentrySpanStatus)status
     id<SentrySpan> spanTracker;
     @synchronized(self.spans) {
         spanTracker = self.spans[spanId];
-        [self.spans removeObjectForKey:spanId];
+        // Hold reference for tracer for longer because automatic
+        // tracer aren't  being referenced by anything else.
```

Comment:
`l`:
```suggestion
        // Hold reference for tracer until the tracer finishes because automatic
        // tracers aren't referenced by anything else.
```